gulpfile.js/build.js

```diff
@@ -167,6 +168,15 @@ async function buildPixi() {
   return sh('npm run build:pixi');
 }
 
+/**
+ * Builds the pixi cloud functions project
+ */
+function buildPixiFunctions() {
+  return sh('npm install', {
+    workingDir: PIXI_ROOT,
+  });
```

Comment:
That's just a stub, right?

Reply:
how do you mean?  It needs to be its own npm module.